simulator/src/test/java/com/hedera/block/simulator/grpc/PublishStreamGrpcClientImplTest.java

```diff
@@ -65,11 +69,28 @@ void streamBlock() {
 
         PublishStreamGrpcClientImpl publishStreamGrpcClient =
                 new PublishStreamGrpcClientImpl(grpcConfig, blockStreamConfig);
-
+        publishStreamGrpcClient.init();
         boolean result = publishStreamGrpcClient.streamBlock(block);
         assertTrue(result);
 
         boolean result1 = publishStreamGrpcClient.streamBlock(block1);
         assertTrue(result1);
     }
+
+    @Test
+    void testShutdown() throws Exception {
+        PublishStreamGrpcClientImpl publishStreamGrpcClient =
+                new PublishStreamGrpcClientImpl(grpcConfig, blockStreamConfig);
+        publishStreamGrpcClient.init();
+
+        Field channelField = PublishStreamGrpcClientImpl.class.getDeclaredField("channel");
+        channelField.setAccessible(true);
+
+        ManagedChannel mockChannel = mock(ManagedChannel.class);
+        channelField.set(publishStreamGrpcClient, mockChannel);
+        publishStreamGrpcClient.shutdown();
```

Comment:
A small nit and for best practice you could do a try-finally on the field visibility setting, up to you:

```suggestion
        ManagedChannel mockChannel = mock(ManagedChannel.class);

        Field channelField = PublishStreamGrpcClientImpl.class.getDeclaredField("channel");
        try {
            channelField.setAccessible(true);
            channelField.set(publishStreamGrpcClient, mockChannel);
        } finally {
            channelField.setAccessible(false);
        }

        publishStreamGrpcClient.shutdown();
```

Reply:
Changed. 👍